icons/database-backup.svg

```diff
@@ -10,9 +10,9 @@
   stroke-linejoin="round"
 >
   <ellipse cx="12" cy="5" rx="9" ry="3" />
-  <path d="M3 12c0 1.18 2.03 2.2 5 2.7" />
-  <path d="M21 5v4.5" />
-  <path d="m12 16 1.27-1.35a4.75 4.75 0 1 1 .41 5.74" />
+  <path d="M3 5v14c0 1.4 3 2.7 7 3" />
+  <path d="M3 12c0 1.2 2 2.5 5 3" />
+  <path d="M21 5v4" />
+  <path d="M13 20c.9 1.2 2.3 2 4 2 2.8 0 5-2.2 5-5 0-1.2-.4-2.3-1.3-3.2-1.8-1.8-4.6-1.8-6.4 0L12 16" />
```

Comment:
A very slight optimization:
```suggestion
  <path d="M13 20a5 5 0 0 0 9-3 4.5 4.5 0 0 0-4.5-4.5c-1.33 0-2.54.54-3.41 1.41L12 16" />
```

Reply:
This results in a shape slightly off, at 10x10.0003px… Sure?

Reply:
Well _without_ the arrow head, yours is 10x9.55 in Figma, my proposal is 10x9.5, hence the optimization. 😄

Reply:
> Well _without_ the arrow head, yours is 10x9.55 in Figma, my proposal is 10x9.5, hence the optimization. 😄

I think this is another Figma vs Illustrator thing… It was 10x9.5 in Illustrator 🙈